frostsnap_core/src/coordinator.rs

```diff
@@ -913,8 +1068,16 @@ impl Gist for Mutation {
             NoncesUsed { .. } => "NoncesUsed",
             ResetNonces { .. } => "ResetNonces",
             NewNonces { .. } => "NewNonces",
-            NewKey(_) => "NewKey",
+            NewAccessStructure { .. } => "NewAccessStructure",
+            NewKey { .. } => "NewKey",
         }
         .into()
     }
 }
+
+// Uniquely identifies an access structure for a particular `appkey`.
+#[derive(Debug, Clone, Copy)]
+pub struct AccessStructureRef {
```

Comment:
Noticed this way of referencing access structures means we don't look them up with some `coordinator.get_appkey_from_access_structure(access_structure_id)`, any reason?

Reply:
ah of course..